radio.py

```diff
@@ -67,41 +68,67 @@
     ['BBC WM', 'http://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/dash/nonuk/dash_low/llnw/bbc_wm.mpd'],
     ['BBC York', 'https://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/dash/nonuk/dash_low/aks/bbc_radio_york.mpd'],
     ['BBC World Service - Internet Schedule', 'http://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/hls/nonuk/sbr_low/llnw/bbc_world_service.m3u8'],
-    ['BBC World Service - English News', 'http://www.bbc.co.uk/worldservice/meta/live/mp3/ennws.pls'],
     ['BBC World Service - Africa', 'http://www.bbc.co.uk/worldservice/meta/live/mp3/enafw.pls'],
     ['BBC Arabic Radio', 'https://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/dash/nonuk/dash_low/llnws/bbc_arabic_radio.mpd'],
 ]
-WIN_TITLE = "BBC radio"
+
+# URL list taken from http://icestreaming.rai.it/status.xsl. Renamed following https://www.raiplayradio.it/
+RAI_STATIONS = [
+    ['RAI Radio 1', 'http://icestreaming.rai.it/1.mp3'],
+    ['RAI Radio 1 Sport', 'http://icestreaming.rai.it/13.mp3'],
+    ['RAI Radio 2', 'http://icestreaming.rai.it/2.mp3'],
+    ['RAI Radio 2 Indie', 'http://icestreaming.rai.it/15.mp3'],
+    ['RAI Radio 3', 'http://icestreaming.rai.it/3.mp3'],
+    ['RAI Radio 3 Classica', 'http://icestreaming.rai.it/5.mp3'],
+    ['RAI IsoRadio', 'http://icestreaming.rai.it/6.mp3'],
+    ['RAI GrParlamento', 'http://icestreaming.rai.it/7.mp3'],
+    ['RAI Radio Kids', 'http://icestreaming.rai.it/11.mp3'],
+    ['RAI Radio Live', 'http://icestreaming.rai.it/10.mp3'],
+    ['RAI Radio Techete\'', 'http://icestreaming.rai.it/9.mp3'],
```

Comment:
This should be "Techetè". What you're using is an alternative spelling trying to simulate accents when only ASCII is available, but we can use proper Unicode here.

Reply:
Seems to be [techete'](https://www.raiplayradio.it/radiotechete/)

Reply:
Trust me, I've been living in Italy for more than 13 years. What you're seeing there is the result of web developers not figuring out how to use Unicode, or not bothering to update old ASCII workarounds after moving the site to Unicode.

Reply:
yeah I know the italian style lol